particle/data/pdgID_to_latex.txt

```diff
@@ -0,0 +1,356 @@
+pdgid,particle,antiparticle
+1,d,d
+2,u,u
+3,s,s
+4,c,c
+5,b,b
+6,t,t
+11,e^-,e^+
+13,\mu^-,\mu^+
+15,\tau^-,\tau^+
+22,\gamma,\gamma
+23,Z^{0},Z^{0}
+24,W^{+},W^{-}
+111,\pi^{0},\pi^{0}
+113,\rho(770)^{0},\rho(770)^{0}
+115,a_{2}(1320)^{0},a_{2}(1320)^{0}
+117,\rho_3(1690)^{0},\rho_3(1690)^{0}
+119,a_4(2040)^{0},a_4(2040)^{0}
+130,K_L^{0},K_L^{0}
+211,\pi^{+},\pi^{-}
+213,\rho(770)^{+},\rho(770)^{-}
+215,a_{2}(1320)^{+},a_{2}(1320)^{-}
+217,\rho_3(1690)^{+},\rho_3(1690)^{-}
+219,a_4(2040)^{+},a_4(2040)^{-}
+221,\eta,\eta
+223,\omega(782),\omega(782)
+225,f_{2}(1270),f_{2}(1270)
+227,\omega_3(1670),\omega_3(1670)
+229,f_4(2050),f_4(2050)
+310,K_S^{0},K_S^{0}
+311,K^{0},K^{0}
+313,K^{*}(892)^{0},K^{*}(892)^{0}
+315,K_{2}^{*}(1430)^{0},K_{2}^{*}(1430)^{0}
+317,K_3^{*}(1780)^{0},K_3^{*}(1780)^{0}
+319,K_4^{*}(2045)^{0},K_4^{*}(2045)^{0}
+321,K^{+},K^{-}
+323,K^{*}(892)^{+},K^{*}(892)^{-}
+325,K_{2}^{*}(1430)^{+},K_{2}^{*}(1430)^{-}
+327,K_3^{*}(1780)^{+},K_3^{*}(1780)^{-}
+329,K_4^{*}(2045)^{+},K_4^{*}(2045)^{-}
+331,\eta^\prime(958),\eta^\prime(958)
+333,\phi(1020),\phi(1020)
+335,f_{2}^\prime(1525),f_{2}^\prime(1525)
+337,\phi_3(1850),\phi_3(1850)
+411,D^{+},D^{-}
+413,D^{*}(2010)^{+},D^{*}(2010)^{-}
+415,D_{2}^{*}(2460)^{+},D_{2}^{*}(2460)^{-}
+421,D^{0},D^{0}
+423,D^{*}(2007)^{0},D^{*}(2007)^{0}
+425,D_{2}^{*}(2460)^{0},D_{2}^{*}(2460)^{0}
+431,D_s^{+},D_s^{-}
+433,D_s^{*+},D_s^{*-}
+435,D_{s2}^{*+},D_{s2}^{*-}
+441,\eta_c(1S),\eta_c(1S)
+443,J/\psi(1S),J/\psi(1S)
+445,\chi_{c2}(1P),\chi_{c2}(1P)
+511,B^{0},B^{0}
+513,B^{*0},B^{*0}
+515,B_{2}^{*0},B_{2}^{*0}
+521,B^{+},B^{-}
+523,B^{*+},B^{*-}
+525,B_{2}^{*+},B_{2}^{*-}
+531,B_s^{0},B_s^{0}
+533,B_s^{*0},B_s^{*0}
+535,B_{s2}^{*0},B_{s2}^{*0}
+541,B_c^{+},B_c^{-}
+551,\eta_b(1S),\eta_b(1S)
+553,\Upsilon(1S),\Upsilon(1S)
+555,\chi_{b2}(1P),\chi_{b2}(1P)
+1112,\Delta(1620)^-,\Delta(1620)^+
+1114,\Delta^-,\Delta^+
```

Comment:
It will be confusing for the reader if the "bar" is not present where needed, e.g. the anti-particle of the Delta- is the Deltabar+. This should not be confused with the Delta+ particle.

I really think the 3rd column should be the full antiparticle name.

Reply:
I think I would rather remove the third column; it's computable from the second column and the extended information.